Documentation/network/servicemesh/ingress.rst

```diff
@@ -107,6 +107,18 @@ Supported Ingress Annotations
      - | Enable websocket passthrough support.
        | Applicable values are ``enabled`` and ``disabled``.
      - ``disabled``
+   * - ``ingress.cilium.io/ssl-passthrough``
+     - | Enable SSL Passthrough mode for this Ingress.
+       | Applicable values are ``enabled`` and ``disabled``, although boolean-style
+       | values will also be accepted.
+       | Note that some conditions apply to SSL Passthrough Ingresses, due to how
+       | TLS Passthrough works:
+       | * A ``host`` field must be set in the Ingress
+       | * Default backends are ignored
+       | * Rules with paths other than ``/`` are ignored
+       | If all the rules in an Ingress are ignored for these reasons, no Envoy config
+       | will be generated and the Ingress will have no effect.
```

Comment:
I'm getting a weird formatting issue with the rendering of the table with your change, compare the current table:

<details><summary>Screenshot 1</summary>

![table1](https://github.com/cilium/cilium/assets/17001771/6307bc05-98be-4fec-8340-f2e3a395a424)
</details>

with the one after your change:

<details><summary>Screenshot 2</summary>

![table2](https://github.com/cilium/cilium/assets/17001771/238ee501-9540-4d21-a21e-f0c09223e283)
</details>

There are some earlier lines that are no longer wrapped and that overflow into the third column. This is probably some bug in how Sphinx renders tables, because my understanding is that these lines should not be wrapped in the first place :shrug:.

Anyway, I don't have a clean fix, so please would you mind wrapping your own description _more_, which seems to keep the earlier lines as they are?

```suggestion
     - | Enable SSL Passthrough mode for this Ingress.
       | Applicable values are ``enabled`` and ``disabled``,
       | although boolean-style values will also be
       | accepted.
       | Note that some conditions apply to SSL
       | Passthrough Ingresses, due to how
       | TLS Passthrough works:
       | * A ``host`` field must be set in the Ingress
       | * Default backends are ignored
       | * Rules with paths other than ``/`` are ignored
       | If all the rules in an Ingress are ignored for
       | these reasons, no Envoy config will be generated
       | and the Ingress will have no effect.
```

Reply:
Done, thanks!